### ISSUE TITLE
GitHub Action airslate-oss/setup-specmatic

### ISSUE THREAD

**Issue by sergeyklay** [specmatic/specmatic#602]
It's not really an issue, but more of an announcement. You can close this topic at any time.

I am pleased to introduce the first GitHub Action to set up a Specmatic environment for use in actions by optionally downloading and caching a version of specmatic by version and adding to `PATH`: https://github.com/marketplace/actions/setup-specmatic-environment

It doesn't do much at the moment, but it's already handy. I plan to actively maintain it for some time to provide additional features. Should you have any question, any remark, or if you find a bug, or if there is something you can't do with the
`setup-specmatic`, please [open an issue](https://github.com/airslate-oss/setup-specmatic/issues).

**Comment by harikrishnan83** [comment]
@sergeyklay Thanks a lot. I took your Github Action for a spin it worked flawlessly and it is really intuitive. It saves a lot of effort by caching the jar properly. Appreciate this very much. We have featured this tool in our [documentation](https://specmatic.in/documentation/continuous_integration.html).

I am yet to try out some of the other features, will keep you posted / [open an issue](https://github.com/airslate-oss/setup-specmatic/issues) if we find any bugs.
